randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -177,17 +177,31 @@ Extra - asset_id: 4
   * Normal Door to The Moat/Door to West Ocean
   > Door to Wrecked Ship Entrance
       Any of the following:
-          # TODO: Check to see if possible with HBJ.
           Grapple Beam or Space Jump
-          # It is possible to jump across by not using full jumps to avoid bonking on the ceiling.
+          Horizontal Infinite Bomb Jump (Expert) and Can Lay Bombs
+          # It is possible to jump across by not using full jumps to avoid bonking on the celing.
           Movement (Intermediate)
           All of the following:
-              # From the sea floor
+              # From the Sea Floor
               Gravity Suit
               Any of the following:
                   Hi-Jump Boots or Wall Jumps (Beginner)
                   # IBJ
                   Vertical Infinite Bomb Jump (Beginner) and Can Lay Bombs
+  > Door from Gravity Suit Room
+      Any of the following:
+          Space Jump
+          All of the following:
+              # Get over and WJ up
+              Wall Jumps (Intermediate)
+              Grapple Beam or Movement (Intermediate)
```

Comment:
Whats the movement for?

Reply:
To jump over to the ledge below the door. Vanilla is Grapple, can be done with just jumps if you don't full jump and have the right momentum.